Automate/Speckle.Automate.Sdk/AutomationContext.cs

```diff
@@ -15,40 +15,41 @@ public class AutomationContext
 {
   public AutomationRunData AutomationRunData { get; set; }
   public string? ContextView => AutomationResult.ResultView;
-  public Client SpeckleClient { get; set; }
+  public Client SpeckleClient { get; private set; }
 
-  private ServerTransport serverTransport;
-  private string speckleToken;
+  private ServerTransport _serverTransport;
+  private string _speckleToken;
 
   // keep a memory transport at hand, to speed up things if needed
-  private MemoryTransport memoryTransport;
+  private MemoryTransport _memoryTransport;
 
   // added for performance measuring
-  private Stopwatch initTime;
+  private Stopwatch _initTime;
 
-  internal AutomationResult AutomationResult { get; set; }
+  internal AutomationResult AutomationResult { get; private set; }
 
   public static async Task<AutomationContext> Initialize(AutomationRunData automationRunData, string speckleToken)
   {
-    var account = new Account
-    {
-      token = speckleToken,
-      serverInfo = new ServerInfo { url = automationRunData.SpeckleServerUrl }
-    };
+    Account account =
+      new()
+      {
+        token = speckleToken,
+        serverInfo = new ServerInfo { url = automationRunData.SpeckleServerUrl }
+      };
     await account.Validate().ConfigureAwait(false);
-    var client = new Client(account);
-    var serverTransport = new ServerTransport(account, automationRunData.ProjectId);
-    var initTime = new Stopwatch();
+    Client client = new(account);
+    ServerTransport serverTransport = new(account, automationRunData.ProjectId);
+    Stopwatch initTime = new();
     initTime.Start();
 
     return new AutomationContext
     {
       AutomationRunData = automationRunData,
       SpeckleClient = client,
-      serverTransport = serverTransport,
-      speckleToken = speckleToken,
-      memoryTransport = new MemoryTransport(),
-      initTime = initTime,
+      _serverTransport = serverTransport,
+      _speckleToken = speckleToken,
+      _memoryTransport = new MemoryTransport(),
+      _initTime = initTime,
       AutomationResult = new AutomationResult(),
     };
```

Comment:
This should have a constructor for it. Out of scope of this PR, we should open a ticket for this.